### PR TITLE
TT-7962 MDCB 2.1.0 release notes

### PR DESCRIPTION
**Pull request by tbuchaillot**
## Related Issue (if applicable)

https://tyktech.atlassian.net/browse/TT-7962
## Description

Adding changelog for MDCB 2.1.0
## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/release-notes/mdcb-2.1.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    parent: "Release Notes"
+weight: 254
+---
+
+## 2.1.0
+Release date: 2023-02-20
+
+### Added
+- Added a new configuration option (`enable_ownership`) that allows MDCB filter APIs by API Ownership. 
+- MDCB works without group id. This means that when an Edge Gateway doesn’t have a group, it will defaults to the `ungrouped` group. This has some fallbacks, as we can’t use the synchroniser for the ungrouped gateways.
```
Comment (sredxny) [inline tyk-docs/content/release-notes/mdcb-2.1.md:14]: I think this line should be part of `Fixed`. As here mostly we add new config options

File: tyk-docs/content/release-notes/mdcb-2.1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  main:
+    parent: "Release Notes"
+weight: 254
+---
+
+## 2.1.0
+Release date: 2023-02-20
+
+### Added
+- Added a new configuration option (`enable_ownership`) that allows MDCB filter APIs by API Ownership.
```
Comment (sredxny) [inline tyk-docs/content/release-notes/mdcb-2.1.md]: worth to describe the option? Or paste a link to the config option description? so people is aware of default value.

**Review by sredxny — DISMISSED** [review]

File: tyk-docs/content/release-notes/mdcb-2.1.md (hunk 1 above)
Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb-2.1.md:14]: @sredxny IMO, it's an added behavior - it was not a bug.

File: tyk-docs/content/release-notes/mdcb-2.1.md (hunk 2 above)
Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb-2.1.md]: @sredxny we never cross-link with the config option. What else would you add?

Comment (letzya) [reply tyk-docs/content/release-notes/mdcb-2.1.md]: we should definitely add links. The main content should be outside the release notes so we don't need to have it twice. Is the option described in the doc?

File: tyk-docs/content/release-notes/mdcb-2.1.md (hunk 1 above)
Comment (letzya) [reply tyk-docs/content/release-notes/mdcb-2.1.md:14]: same comment - is this in the docs as well? we have a problem with the pump and MDCB of writing the content only in the release notes and then people might not find it

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/release-notes/mdcb-2.1.md (hunk 2 above)
Comment (sredxny) [reply tyk-docs/content/release-notes/mdcb-2.1.md]: @tbuchaillot usually in the config page we described the config. how is useful, default value and if applies: how the behavior can change when combined with other options. Eg: synchronizer only works when we have enabled the sync in MDCB and worker gw.... So, a link to the config description is useful

Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb-2.1.md]: added link

**Review by letzya — APPROVED** [review]

**Pull request merged**